pycbc/detector.py

```diff
@@ -270,18 +270,81 @@ def time_delay_from_detector(self, other_detector, right_ascension,
                                              declination,
                                              t_gps)
 
-    def project_wave(self, hp, hc, longitude, latitude, polarization):
+    def project_wave(self, hp, hc, ra, dec, polarization,
+                     method='lal',
+                     reference_time=None):
         """Return the strain of a waveform as measured by the detector.
         Apply the time shift for the given detector relative to the assumed
         geocentric frame and apply the antenna patterns to the plus and cross
         polarizations.
+
+        Parameters
+        ----------
+        hp: pycbc.types.TimeSeries
+            Plus polarization of the GW
+        hc: pycbc.types.TimeSeries
+            Cross polarization of the GW
+        ra: float
+            Right ascension of source location
+        dec: float
+            Declination of source location
+        polarization: float
+            Polarization angle of the source
+        method: str, Optional
+            The method to use for projecting the polarizations into the
+            detector frame. Default is 'lal'.
```

Comment:
Could you list the options here? I think the numpy syntax for parameters that take a set of options is to do:
```
method : {'lal', 'constant', 'vary_polarization'}
```